apps/frontend/src/libs/components/map-location-field/map-location-logic/map-location-logic.tsx

```diff
@@ -26,6 +26,12 @@ const MapLocationLogic = ({ location, onLocationChange }: Properties): null => {
 
 	const handleAddMarker = useCallback(
```

Comment:
i think in this case it's better to rename it to set
```suggestion
	const handleSetMarker = useCallback(
```

Reply:
Agree, done
@liza-veis